pkgs/build-support/cc-wrapper/default.nix

```diff
@@ -120,6 +120,17 @@ let
          null)
     else "";
 
+  parseResponseFile = if buildPackages.stdenv.cc or null != null && buildPackages.stdenv.cc != "/dev/null"
+  then buildPackages.stdenv.mkDerivation {
+    name = "parse-response-file";
+    src = ./parseResponseFile.c;
+    buildCommand = ''
+      # Make sure the output file doesn't refer to the input nix path
```

Comment:
We got a runtime dependency on the source file, probably because `__FILE__` in the assert expansion.